indexer/api/routes/deposits.go

```diff
@@ -10,16 +10,16 @@ import (
 )
 
 type DepositItem struct {
-	Guid string `json:"guid"`
-	From string `json:"from"`
-	To   string `json:"to"`
-	// TODO could consider OriginTx to be more generic to handling L2 to L2 deposits
-	// this seems more clear today though
-	Tx      Transaction `json:"Tx"`
-	Block   Block       `json:"Block"`
-	Amount  string      `json:"amount"`
-	L1Token TokenInfo   `json:"l1Token"`
-	L2Token TokenInfo   `json:"l2Token"`
+	Guid           string `json:"guid"`
+	From           string `json:"from"`
+	To             string `json:"to"`
+	Timestamp      uint64 `json:"timestamp"`
+	L1TxHash       string `json:"L1TxHash"`
+	L2TxHash       string `json:"L2TxHash"`
+	L1BlockHash    string `json:"Block"`
```

Comment:
technicalllyyy we could also return the L2BlockHash (only if the deposit has been included on L2, otherwise null) but checking if the deposit message has been relayed on L2.

But if there isn't a huge need for this, we can avoid adding it unless it's asked ofr